src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -96,6 +96,7 @@ export class Dropdown extends Disposable {
 
 	private _onFocus = this._register(new Emitter<void>());
 	public onFocus: Event<void> = this._onFocus.event;
+	private _widthControlElement!: HTMLElement;
```

Comment:
you shouldn't need to assert this (`!`) since you define it in the constructor. you can instead make it readonly (probably best)

Reply:
This is what happens when I copy code :D 

I'll make it readonly, makes much more sense.... I miss `final` from Java now.